data.ts

```diff
@@ -0,0 +1,46 @@
+import { Delivery } from "./types/delivery";
+export const dataDevlivries: Delivery[] = [
+  {
+    id: "1",
+    amount: 200,
+    deadline: new Date("2018-05-30T00:00:00.000Z"),
+    orders: [
+      {
+        id: "1sdasdas21112",
+        userId: "asdausdhasu21",
+        status: "pending",
+        createdAt: new Date("2018-05-30T00:00:00.000Z"),
+        updatedAt: new Date("2018-05-30T00:00:00.000Z"),
+        deliveryId: 1,
+        catagories: [
+          {
+            name: "בשר",
```

Comment:
Regarding Hebrew in general, you're going to get English names and IDs from the server. I think we'll need to create a mapping from English names/IDs to Hebrew names.  Need to think about it.

Reply:
Bad idea, NVM